SYCL/Sampler/unnormalized-clamp-linear-float.cpp

```diff
@@ -2,7 +2,8 @@
 // RUN: %HOST_RUN_PLACEHOLDER %t.out %HOST_CHECK_PLACEHOLDER
 // RUN: %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
 // RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
-// XFAIL: cuda || (level_zero && windows)
+// Temporary disabled on level_zero && windows (#204)
+// UNSUPPORTED: cuda || (level_zero && windows)
```

Comment:
This change disables execution of the test on CUDA. 
Current version runs the test, but expects non-zero return code. New version doesn't run the test on CUDA at all.

Reply:
Right. Fixed.